manager/manager/utils.py

```diff
@@ -505,6 +505,57 @@ def jira_ticket(request_data):
     )
 
 
+def update_time_loss(jira_id: int, add_time_loss: float = 0.0) -> Response:
+    """Connect to the Rubin Observatory JIRA Cloud REST API to
+    update a jira ticket's specific field
+
+    Params
+    ------
+    jira_id: int
+        Jira ID
+
+    Returns
+    -------
+    Response
+        define response here
```

Comment:
Can you perhaps describe the Response same as the next case, `jira_comment`? something like:
```suggestion
        The response and status code of the request to the JIRA API
```